src/poetry/console/commands/build.py

```diff
@@ -114,6 +123,9 @@ def _get_builder(self) -> Callable[..., None]:
 
         return self._build
 
+    def _has_build_backend_defined(self) -> bool:
+        return "build-backend" in self.poetry.pyproject.data.get("build-system", {})
```

Comment:
Maybe move that inline as we will remove it right after the next release?

Reply:
I think there is no need for it. We will either revert this commit completely or we can reuse it, to still emit a warning if falling back happens.

Reply:
/shrug okay